server/scratchy_server/app.py

```diff
@@ -38,31 +39,13 @@
 db_scratchy.init_app(app)
 api = Api(app)
 
+Res =    ( RoomRes,   UserRes,   MessageRes,   AllRoomRes,   NoIdUserRes,   NoIdMessageRes)
+ResStr = ("RoomRes", "UserRes", "MessageRes", "AllRoomRes", "NoIdUserRes", "NoIdMessageRes")
+ResUrl = ("/api/room/<string:roomId>", "/api/user/<string:userId>", "/api/message/<string:messageId>", "/api/room", "/api/user", "/api/message")
```

Comment:
I would prefer a data structure that linked {Ressource, RessourceName, RessourcePath}.
Let's say you have 50 ressources and you need to correct MessageRes URL, with your implementation you need to find the index of MessageRes in Res tuple and then change the URL in ResUrl at that index.

We not using objects ? Like this it would be more explicit and understandable : 
```python
ressources = [
  { "name": "RoomRes",  "ressource": RoomRes,   "url":   "/api/room/<string:roomId>" },
  ....
]
```